stories/index.stories.js

```diff
@@ -173,3 +174,42 @@ storiesOf('Modal', module)
     </div>
 
   )));
+
+storiesOf('Pagination', module)
+  .add('Default', withInfo('')(() => (
+    <div>
+      <Pagination>
+        <Pagination.First />
+        <Pagination.Prev />
+        <Pagination.Ellipsis />
+
+        <Pagination.Item>{10}</Pagination.Item>
+        <Pagination.Item>{11}</Pagination.Item>
+        <Pagination.Item active>{12}</Pagination.Item>
+        <Pagination.Item>{13}</Pagination.Item>
+        <Pagination.Item disabled>{14}</Pagination.Item>
```

Comment:
One potential issue I saw is that the text color of the disabled one is darker (or has a higher perceptible value) than the non-disabled ones, so it stood out more and didn't register as "disabled" to me.

Reply:
In the dropdown PR you use a lighter shade of purple for the disabled button text. Perhaps the same should be used here?